ext/settings.html

```diff
@@ -1838,7 +1838,7 @@ <h1>Yomitan Settings</h1>
                 </p>
             </div>
         </div>
-        <div class="settings-item settings-item-button" data-modal-action="show,anki-cards"><div class="settings-item-inner">
+        <div class="settings-item settings-item-button" data-modal-action="show,anki-cards" id="configure-anki-card-format-main-settings-entry"><div class="settings-item-inner">
```

Comment:
maybe you can querySelector on `[data-modal-action="show,anki-cards"]` to avoid this id